### ISSUE TITLE
Error during quantized  inference for network containing BatchNorm Layer

### ISSUE THREAD

**Issue by uu-praveeng** [ARM-software/armnn#345]
Hi,

I have created the mobilenetv2 network without conv+batchNorm fusion. I have created the FP32 model quantized using the single image .When I ran the inference on the image I am facing below error 
```
ArmNN v20191100

Quantization parameters for Softmax layer (Scale: 0.00344638 and Offset: 0) are incorrect and have been updated to Scale: 0.00390625 and Offset: 0
An error occurred when preparing the network workloads:  BatchNormalizationQueueDescriptor: input and output must have the same quantization space, input has offset 160 and scale 0.0226134, output has offset 146 and scale 0.107056
terminate called after throwing an instance of 'std::out_of_range'
  what():  _Map_base::at
Abort

```

 Could  you guys look into this issue. 
I have attached the below files for your reference
1. Mobilenet.cpp - for FP32 inference and armnn model creation
2. img.txt for FP32 inference
3. img.raw , csv file used for quantization
4. Quantized and .FP32 models
5. MobilenetINT8.cpp used for INT8 inference.

[MobilenetV2withBatchNorm.zip](https://github.com/ARM-software/armnn/files/4232185/MobilenetV2withBatchNorm.zip)
Thank you,
Praveen.

**Comment by MikeJKelly** [comment]
Hi Praveen,

just looking at this now, I'll let you know how it's progressing.

Best regards,
Mike

**Comment by MikeJKelly** [comment]
Hi Praveen, 

the error should be fixed by https://review.mlplatform.org/c/ml/armnn/+/2837 I'm looking into the Softmax warning now.

**Comment by uu-praveeng** [comment]
Hi @MikeJKelly ,

Thank you for looking into this . This really helps. 
Could you let us know in which release this fix is expected.

Thank you,
Praveen.

**Comment by MikeJKelly** [comment]
Hi Praveen,

so sorry for the delay, this is on the master branch now and will be in the 20.05 release.
